app.js

```diff
@@ -58,28 +64,23 @@ app.use(errorLogger);
 
 app.use(errors());
 
-app.use((err, req, res, next) => {
-  const isValidationError = err.message.indexOf('ValidationError');
-
-  if (err.message && isValidationError) {
-    res.status(400).send({ message: err.message });
-  } else {
-    const { statusCode = 500, message } = err;
-
-    res
-      .status(statusCode)
-      .send({
-        message: statusCode === 500
-          ? 'На сервере произошла ошибка'
-          : message,
-      });
+app.use('/', (err, req, res, next) => {
+  if (err) {
+    next(err);
```

Comment:
Нужно исправить: не работает обработчик несуществующего роута.
В _Express_ у обычных обработчиков 3 аргумента. В обычные обработчики `err` никогда не попадёт:
```suggestion
app.use('/', (req, res, next) => {
  next(new NotFoundError('Запрашиваемый ресурс не найден'));
```

Остаётся только подключить модуль.